packages/botonic-nlp/src/preprocess/vocabulary-generator.ts

```diff
@@ -1,9 +1,13 @@
-import { Sample } from '../parser/types'
+import { Sample } from '../dataset/types'
```

Comment:
i would try to generate the types with the following command and include them in `src` (not needed in this PR)
`npx typescript src/**/*.js --declaration --allowJs --emitDeclarationOnly --outDir types`.
But it will allow you to keep the types with the same file structure decoupled from the src code. Then, when distributed you can add a reference in `package.json` to where the types are stored. (see https://github.com/hubtype/botonic/pull/1333/files#diff-6382927619ccd22effd05dfb119d6a2d8f282b4f4b323e850ce7ffc9a81261feR52)